js/everything.js

```diff
@@ -474,7 +473,13 @@ $(function() {
             );
         }
     }
-
+    if ( /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent) )
```

Comment:
This needs to be moved up so it's the first thing done in the onready listener, before  the `// Keen.io` line.